src/vcf/assembly_checker.cpp

```diff
@@ -121,7 +121,7 @@ namespace ebi
           if (rc.get()) {
               fetch_contig_on_demand = true;
           }
-
+          std::unordered_set<std::string> downloadedContigs;
```

Comment:
why do we need this set if there are also methods in IFasta to ask if the contig is available, and download it automatically if not?

Reply:
Indeed the fasta utility class provides `count` method which can be used to check if an atempt for downloading a contig has been made. Removed in d599e3b.